mathics/builtin/numbers/calculus.py

```diff
@@ -58,6 +66,33 @@
 SymbolIntegrate = Symbol("Integrate")
 
 
+def get_accuracy_and_prec(opts: dict, evaluation: "Evaluation"):
+    """
+    Looks at an opts dictionary and tries to determine the numeric values of
+    Accuracy and Precision goals. If not available, returns None.
+    """
+    acc_goal = opts.get("System`AccuracyGoal", None)
+    if acc_goal:
+        acc_goal = apply_N(acc_goal, evaluation)
+        if acc_goal is SymbolAutomatic:
+            acc_goal = from_python(9.0)
```

Comment:
Where does the number 9.0 come from?

Reply:
In WMA, `Chop` by default round numbers to 10 significant digits. Here I just look for a number around 10 that provides results accurate enough to pass the tests. In any case, for a final version, should let to the different methods that define the defaults.